### DISCUSSION TITLE
Integrate PulseChain (Eth fork in development) to Wormhole

### DISCUSSION THREAD

**Discussion by jv2pt0** [wormhole-foundation/wormhole#785] [Feature requests]
Along with current bridges in the ecosystem, it would be great if the wormhole can support PulseChain as well.

PulseChain is forking the ethereum blockchain including the full system state and is currently operating on their testnet in advance of an upcoming mainnet launch. Due to the forking of the system state, this chain is primed to explode onto the DeFi space and the connections to other blockchains (notably Terra & UST) will be extremely valuable. 

I would like to coordinate with Wormhole to build out an extension to the PulseChain ecosystem and request developers to look into this.

**Comment by priyanshu9899** [comment]
.
